### ISSUE TITLE
UX: encourage user contributions in the Recipes Catalog

### ISSUE THREAD

**Issue by MariaLeonova** [containers/podman-desktop-extension-ai-lab#1629]
### Is your enhancement related to a problem? Please describe

We would like to include a way in the recipe catalog to indicate to the end-users and the community, where they can find the catalog, how they can expand it with their own catalog and eventually contribute to the upstream catalog.

### Describe the solution you'd like

UX will figure out a way to make this experience most pleasurable.

**Comment by MariaLeonova** [comment]
@slemeur @jeffmaury is there a documented existing experience for the contributions?

**Comment by slemeur** [comment]
Yes, it's documented here: https://github.com/containers/ai-lab-recipes/blob/main/CONTRIBUTING.md 
(but it does not appear on this repo)

**Comment by MariaLeonova** [comment]
I would suggest as **step 1** to add a button that would open a modal. The modal could link to the guidelines: https://github.com/containers/ai-lab-recipes/blob/main/CONTRIBUTING.md

I think this could be a good step one, and if eventually we would like to add an ability to add recipes through the UI, that could, too, be possible.
![Image](https://github.com/user-attachments/assets/fc9f27ef-779c-4a38-ad31-191e72c908a2)
![Image](https://github.com/user-attachments/assets/c8bc999a-70f5-44fe-93db-f3caf892c466)

**Comment by axel7083** [comment]
After some discussion in the UX call, we made the link with https://github.com/containers/podman-desktop-extension-ai-lab/issues/1095 and how it would be nice to allow user to import recipe from the UI.

The idea would be to have the documentation inside the Import page, in addition of the possibility to import a recipe from either a Git repository or a local folder.

**Comment by slemeur** [comment]
@axel7083 : Yes sure, but that's out of the scope for this ticket. In this exact ticket, we don't need to do more than 

Import recipe https://github.com/containers/podman-desktop-extension-ai-lab/issues/1095, is something we'll be scheduling later on.

**Comment by MariaLeonova** [comment]
> The proposed button "New Recipe" feels looking more as a way for the end-user to import a recipe in the list, not about contributing. By seeing this button, are end-user going to understand that the recipe catalog is coming from an opensource project and people could be contributing to this project?
> 

There is really no way to tell without showing it to the users. I would suppose that they would get none of that by just looking at a button. 

> There are two options that we could probably explore, and would be interested to get your opinion on which one could be the most efficient for the end-user.
> 
> 1. Bellow the title of the page, we could have an introduction summary about the Recipe. We could explain what they are, where they are coming from and entice the users to contribute/request new recipes.
> 2. At the bottom of the list, we could have an explanation line.

I would go with option 1, ad the button would then come after the said intro.
Can you please write a couple sentence that can be used in this context?

**Comment by MariaLeonova** [comment]
@slemeur @axel7083  please take a look:

![Image](https://github.com/user-attachments/assets/ef04edfa-eed9-4901-8f39-45cccb6f03c5)

**Comment by axel7083** [comment]
@MariaLeonova we already have a pretty big banner for _gpu promotion_ see bellow

![Image](https://github.com/user-attachments/assets/b561b922-43be-4ed6-8883-9dd594f92583)

This banner can be hidden but is visible by default. If we have both we might not see the recipe at all.

As solution we could say that the GPU promotion banner could be placed only in specific pages, currently it is visible on all.

**Comment by axel7083** [comment]
We could move the GPU banner somewhere else also

![Image](https://github.com/user-attachments/assets/273571b8-df37-47c3-bb5a-071f7e56c3a9)

somewhere less intrusive

**Comment by MariaLeonova** [comment]
> [@MariaLeonova](https://github.com/MariaLeonova) we already have a pretty big banner for _gpu promotion_ see bellow
> 
> ![Image](https://github.com/user-attachments/assets/b561b922-43be-4ed6-8883-9dd594f92583)
> 
> This banner can be hidden but is visible by default. If we have both we might not see the recipe at all.
> 
> As solution we could say that the GPU promotion banner could be placed only in specific pages, currently it is visible on all.

Wow! Why don't I have that, I wonder?

**Comment by axel7083** [comment]
> Wow! Why don't I have that, I wonder?

Look at the Podman configuration (`settings.json`), you may have clicked once on `Don't display anymore` and this has been hidden forever.

**Comment by MariaLeonova** [comment]
Huh, okay. I do wonder about that banner though - the first message is great, makes me want to enable right away (which I most likely did and forgot). 
But the second line there is confusing, and reads more like a status (GPU support is not enabled) and the 2 buttons are both primary, which probably should not happen. 

I also wonder where could users read more about this feature, as I imagine some might be curious.

**Comment by MariaLeonova** [comment]
@slemeur please take a look:

![Image](https://github.com/user-attachments/assets/51bdc246-a69d-487d-878a-e78e809f9a46)

**Comment by slemeur** [comment]
Thanks @MariaLeonova. In term of UX, the proposal is working, but I think we can review the wording.
Could you please paste the text you have here in the issue, so we can edit and do other proposals?

**Comment by MariaLeonova** [comment]
Yes! Here it is:

A "recipe" is a pre-packaged and deployable demo intended to help developers get up and running with AI-based applications as fast as possible. All of our recipes are built to run with AI Lab. They should also contain sufficient code and documentation to be run manually so that developers are able to update and customize them to their own needs.

Recipes are currently organized by category: audio/, computer_vision/, multimodal/, natural_language_processing/. You should also be able to make your own category, if needed.

**Comment by slemeur** [comment]
> A "recipe" is a pre-packaged and deployable demo intended to help developers get up and running with AI-based applications as fast as possible. All of our recipes are built to run with AI Lab. They should also contain sufficient code and documentation to be run manually so that developers are able to update and customize them to their own needs.

> Recipes are currently organized by category: audio/, computer_vision/, multimodal/, natural_language_processing/. You should also be able to make your own category, if needed.

Recipes help you explore and get started with number of core AI use cases like chatbots, code generators, text summarizers, agents and more. Each recipe comes with detailed explanations and runnable source code compatible with various large language models (LLMs). 
Recipes are organized into categories: audio, computer_vision, multimodal, natural_language_processing.

Want to contribute more AI applications? The catalog is open source and available on GitHub!

[Browse Recipes Repository]

**Comment by MariaLeonova** [comment]
Updated - please see. I added an article in front of `number of core uses cases`:

![Image](https://github.com/user-attachments/assets/5b04d274-2b02-40f7-bd0f-1c30fbba2d25)

**Comment by MariaLeonova** [comment]
@slemeur please review!

**Comment by slemeur** [comment]
Thanks @MariaLeonova for the ping on the ticket. 

I'm fine with the mockup here: https://github.com/containers/podman-desktop-extension-ai-lab/issues/1629#issuecomment-2636170170

@jeffmaury : Can you validate from Eng pov?

**Comment by MariaLeonova** [comment]
@jeffmaury can this be closed? Or is there anything else needed for this ticket?

**Comment by MariaLeonova** [comment]
@jeffmaury @slemeur can this be closed? Is there an implementation ticket?
